src/prolly/map.rs

```diff
@@ -537,4 +616,100 @@ mod tests {
         )
         .await;
     }
+
+    macro_rules! map(
```

Comment:
I think it would be more clear to name this macro `prolly_map` since we have another macro we use in tests which is `hash_map!`.

Reply:
+1